src/common/timer/interaction-timer.js

```diff
@@ -71,5 +75,6 @@ export class InteractionTimer extends Timer {
   //   if (!this.remainingMs || !this.isValid()) return
   //   this.timer = this.create(this.cb, this.remainingMs)
   //   this.remainingMs = undefined
+  //   this.onResume?.()
```

Comment:
If this is not being used, maybe we remove this commented out code and remove the `opts.onResume` in the constructor.

Reply:
might need to be renamed, but it is an event that is needed in the session replay feature, to know that a session is "unpaused"